bangazonapi/views/profile.py

```diff
@@ -61,7 +61,7 @@ def list(self, request):
             }
         """
         try:
-            current_user = Customer.objects.get(user__id=4)
+            current_user = Customer.objects.get(user=request.user)
```

Comment:
How do you know this is an authenticated user?

Reply:
I'm not sure I understand? It will only get the user info if the token in the request matches one in the database. I suppose you could use an if statement to check `request.user.is_authenticated`, but in testing I get the same results with that as the above code. What am I missing?

Reply:
Take a look at how the code is getting the authenticated user on here: https://github.com/nss-evening-cohort-12/bangazon-api-jshearon/blob/main/bangazonapi/views/cart.py#L110
What you have technically works as long as the user sends a token. And if they didn't they would get an unauthorized error anyway because of the auth settings in the settings.py file. 
It's pretty common to use the same approach for getting the user from the request (or other things like that) to keep the code consistent in case things break.

Reply:
Have re-pushed to address this change request